### PR TITLE
Add checkpoint_store test

### PR DESCRIPTION
**Pull request by LilyFaFa**

### CODE REVIEW

File: pkg/hyper/checkpoint_store_test.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
Comment (resouer) [inline pkg/hyper/checkpoint_store_test.go]: This boilerplate is wrong, it's 2017 now.

**Comment by resouer** [comment]
@LilyFaFa Thank you very much for contribution. Please sign CNCF CLA first so you will become a official contributor in Kubernetes community.

**Comment by resouer** [comment]
Please fix the CI, it is saying: Boilerplate missing or errored in: ././pkg/hyper/checkpoint_store_test.go.

**Comment by LilyFaFa** [comment]
@resouer Thanks for your time, I think this time should be ok.

**Comment by feiskyer** [comment]
@LilyFaFa Thanks. LGTM

**Pull request merged**